db/schema.rb

```diff
@@ -17,6 +17,8 @@
   enable_extension "plpgsql"
   enable_extension "intarray"
   enable_extension "pg_trgm"
+  enable_extension "hstore"
```

Comment:
yeah i normally do a `rake db:migrate:reset RAILS_ENV=true` but it'd be nice to not have to worry about this. I think it's cause the extensions exist on the staging db (perhaps an old migration) so the schema dumper reads it and adds it to the the dump.